Jenkinsfile

```diff
@@ -32,10 +32,5 @@ try {
 catch (Exception e) {
     echo 'Failure encountered'
 
-    node("ec2-stateful") {
```

Comment:
potentially, and also has the risk of filling the disk over time. in the absence of artifact collection or log shipping though I can't see a quick way to preserve the logs for analysis. an alternate approach to collecting the logs might be to have the error handler try and find and display the associated logs in the build output.